scripts-available/CDB_Overviews.sql

```diff
@@ -284,21 +292,21 @@ AS $$
     IF class_info.relpages < 2 OR fraction > 0.5 THEN
       -- We'll avoid possible CDB_RandomTids problems
       EXECUTE Format('
-        CREATE TABLE %I AS SELECT * FROM %s WHERE random() < %s;
-      ', overview_rel, reloid, fraction);
+        %s SELECT * FROM %s WHERE random() < %s;
+      ', creation_clause, reloid, fraction);
     ELSE
       num_samples := ceil(class_info.reltuples*fraction);
       EXECUTE Format('
-        CREATE TABLE %4$I.%1$I AS SELECT * FROM %2$s
+        %1$s SELECT * FROM %2$s
           WHERE ctid = ANY (
             ARRAY[
               (SELECT CDB_RandomTids(''%2$s'', %3$s))
             ]
           );
-      ', overview_rel, reloid, num_samples, schema_name);
+      ', creation_clause, reloid, num_samples);
     END IF;
 
-    RETURN Format('%I.%I', schema_name, overview_rel)::regclass;
+    RETURN Format('%s', overview_table_name)::regclass;
```

Comment:
I created a a variable which is compounded by schema_name and overview_rel. You can see it [here](https://github.com/CartoDB/cartodb-postgresql/pull/302/files/09309cc100b99e14513da13ae4de9dc242301723#diff-f3f06dde01c50fc3d5586076be12f211R280)